components/MessageOverlay.tsx

```diff
@@ -46,6 +47,18 @@ export const MessageOverlay: React.FC<MessageOverlayProps> = props => {
               {props.message}
             </Text>
           )}
+          {props.failedVcs &&
```

Comment:
Don't Pass VC related details here.
Text for MessageOverlay should be passed from MyVcsTab. MessgaeOverlay will just render it.